src/main/java/ViewModel/playlistViewModel.java

```diff
@@ -1,4 +1,44 @@
 package ViewModel;
-
+import Presenter.playlistPresenter;
 public class playlistViewModel {
+    private playlistPresenter playlistPresenter;
+    private String outputMessage;
+
+    public playlistViewModel(){
+        // get necessary information from View to construct a new playlist
+        //TODO: This is the controller, just pass this to the controller, move to controller
+        //TODO: Create controller once in the view as an instance variable / pass as a parameter in main
```

Comment:
This is the method Gries likes, since he really focuses on dependency injection. However, we've been created the controllers and presenters inside of the view model, which is definitely allowed according to clean architecture. I've also personally asked Prof. Gries about this and he said that it is fine to do this too, he just prefers dependency injection. 

I think we should stick to constructing our controllers and presenters inside of the view model, just to keep our code consistent, even though it introduces some hard dependencies. If we have time at the end of development, we can take a step back and see if we can apply dependency injection anywhere to make things cleaner, but for now we should be consistent.